test/parallel/test-c-ares.js

```diff
@@ -85,7 +85,8 @@ dns.lookup('::1', common.mustCall((error, result, addressType) => {
 // Windows doesn't usually have an entry for localhost 127.0.0.1 in
 // C:\Windows\System32\drivers\etc\hosts
 // so we disable this test on Windows.
-if (!common.isWindows) {
+// IBM i reports `ENOTFOUND` when get hostname by address 12.0.0.1
```

Comment:
Can you refer to the platform consistently to help in searching. I think IBMi was decided on in the previous PR.